### PR TITLE
Nightly build: Async message replies for playlist and ping as well as…

### PR DESCRIPTION
**Pull request by GeorgeGo**
… chris reaction and merge noah PR

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -1,15 +1,47 @@
...
+    // TODO: return latency
+    // TODO: wrap commands with delete and delete_message
+    if (cmd === 'ping') {
+        msg.delete(config.delete_timeout);
+        msg.reply('pong').then(delete_message);
+    }
+
+    if (cmd === 'playlists') {
+        msg.delete(config.delete_timeout);
+        playlists = await get_playlists();
```
Comment (noahgribbin) [inline bot.js]:
```suggestion
        playlists = await fs.readdir('./playlists/');
```

**Review by noahgribbin — COMMENTED** [review]
Consider removing getPlaylist helper function. Convert to one liner.

File: bot.js (hunk 1 above)
Comment (GeorgeGo) [reply bot.js]: wait hold up

**Comment by GeorgeGo** [comment]
Alright I think I fixed the major issues.

**Comment by GeorgeGo** [comment]
@noahgribbin

**Comment by GeorgeGo** [comment]
Alright I think I fixed the major issues. @noahgribbin

**Pull request merged**